waspc/cli/src/Wasp/Cli/Command/Require.hs

```diff
@@ -100,3 +76,43 @@ instance Requirable InWaspProject where
           ( "Couldn't find wasp project root - make sure"
               ++ " you are running this command from a Wasp project."
           )
+
+data DbConnectionEstablishedFromOutDir = DbConnectionEstablishedFromOutDir deriving (Typeable)
+
+-- TODO(carlos): Modify this requirement to work in `.wasp/build` as well (#2858)
```

Comment:
FYI this comment is a standalone comment -> it is not attached to any code, in a way.

If you wanted to make it reference the code below (which I thikn you did), then you should make it start with `-- |` -> this makes it a "declaration comment". Check out Haddock (like jsdoc but for Haskell) docs here: https://haskell-haddock.readthedocs.io/latest/markup.html#documenting-a-top-level-declaration .